validation.py

```
@@ -0,0 +1,681 @@
+        'linkedin': ['linkedin', 'li'],
+        'facebook': ['facebook', 'fb'],
+        'instagram': ['instagram', 'ig'],
+        'tiktok': ['tiktok', 'tt'],
+        'youtube': ['youtube', 'yt'],
+        'reddit': ['reddit', 'red'],
+        'medium': ['medium', 'med'],
+        'blog': ['blog', 'website', 'web'],
+        'press_release': ['press_release', 'press', 'pr'],
+        'email': ['email', 'mail'],
+        'newsletter': ['newsletter', 'news']
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)